From 40360803622c180747096aa2f165a02fef3628a5 Mon Sep 17 00:00:00 2001
From: Komal Bajaj <komal.bajaj@oss.qualcomm.com>
Date: Mon, 3 Nov 2025 16:53:11 +0530
Subject: [PATCH] soc: qcom: socinfo: Add SoC ID for QCS6490

Add SoC ID table entry for Qualcomm QCS6490.

Signed-off-by: Komal Bajaj <komal.bajaj@oss.qualcomm.com>
Link: https://lore.kernel.org/r/20251103-qcs6490_soc_id-v1-2-c139dd1e32c8@oss.qualcomm.com
Signed-off-by: Bjorn Andersson <andersson@kernel.org>
[QWRT: also backport QCOM_ID_QCS6490 define from 7.0.11]
diff -Naur /tmp/qcom-socinfo-qcs6490.6tsxS5/orig/drivers/soc/qcom/socinfo.c /tmp/qcom-socinfo-qcs6490.6tsxS5/new/drivers/soc/qcom/socinfo.c
--- a/drivers/soc/qcom/socinfo.c	2026-07-01 18:53:13.379230126 +0800
+++ b/drivers/soc/qcom/socinfo.c	2026-07-01 18:53:13.384929488 +0800
@@ -415,6 +415,7 @@
 	{ qcom_board_id(SC7280) },
 	{ qcom_board_id(SC7180P) },
 	{ qcom_board_id(QCM6490) },
+	{ qcom_board_id(QCS6490) },
 	{ qcom_board_id(SM7325P) },
 	{ qcom_board_id(IPQ5000) },
 	{ qcom_board_id(IPQ0509) },
diff -Naur /tmp/qcom-socinfo-qcs6490.6tsxS5/orig/include/dt-bindings/arm/qcom,ids.h /tmp/qcom-socinfo-qcs6490.6tsxS5/new/include/dt-bindings/arm/qcom,ids.h
--- a/include/dt-bindings/arm/qcom,ids.h	2026-07-01 18:53:13.378088511 +0800
+++ b/include/dt-bindings/arm/qcom,ids.h	2026-07-01 18:53:13.383122480 +0800
@@ -240,6 +240,7 @@
 #define QCOM_ID_SC7280			487
 #define QCOM_ID_SC7180P			495
 #define QCOM_ID_QCM6490			497
+#define QCOM_ID_QCS6490			498
 #define QCOM_ID_SM7325P			499
 #define QCOM_ID_IPQ5000			503
 #define QCOM_ID_IPQ0509			504
